data/settings.js

```diff
@@ -2102,6 +2102,107 @@ More languages can be obtained from:
 * [GitHub - Stopwords ISO](https://github.com/stopwords-iso/).`
 	},
 
+	language_tokenizer_address_token_maxlen: {
+		plugin: 'fts',
+		tags: [ 'language-tokenizer-email-address' ],
+		values: setting_types.UINT,
+		default: 250,
+		seealso: [ '[[link,fts_tokenization]]' ],
+		text: `
+Maximum length of token, before an arbitrary cut off is made.`
+	},
+
+	language_tokenizer_generic_algorithm: {
+		plugin: 'fts',
+		tags: [ 'language-tokenizer-generic' ],
+		values: setting_types.ENUM,
+		values_enum: [ 'simple', 'tr29' ],
+		default: 'simple',
+		seealso: [ '[[link,fts_tokenization]]' ],
+		text: `
+It defines the method for looking for word boundaries.
```

Comment:
Would be nicer simply as `Defines the ...`

Also "for looking for" sounds weird. Maybe "for finding word boundaries"?